InsertData/testsrc/InsertDataTest.java

```diff
@@ -0,0 +1,190 @@
+import static com.mongodb.client.model.Filters.in;
+import static org.junit.Assert.*;
+
+import org.bson.Document;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.result.DeleteResult;
+
+import org.json.JSONException;
+import org.json.simple.parser.ParseException;
+import org.junit.Test;
+
+
+public class InsertDataTest {
+
+	String CONNECTION = "mongodb+srv://m001-student:student123#@sandbox-trhqa.mongodb.net/test";
```

Comment:
Connection String is the common word used across databases to specify database information. By changing the variable name, the code could more easily be understood by a reader.

Reply:
changes made